src/include/migraphx/apply_alpha_beta.hpp

```diff
@@ -0,0 +1,77 @@
+#ifndef MIGRAPHX_GUARD_MIGRAPHX_APPLY_ALPHA_BETA_HPP
+#define MIGRAPHX_GUARD_MIGRAPHX_APPLY_ALPHA_BETA_HPP
+
+#include "migraphx/errors.hpp"
+#include <migraphx/instruction.hpp>
+#include <migraphx/instruction_ref.hpp>
+#include <migraphx/make_op.hpp>
+#include <migraphx/common.hpp>
+#include <migraphx/module.hpp>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+template <typename T = float>
+instruction_ref insert_apply_alpha_beta(module& m,
+                                        instruction_ref pos,
+                                        const std::vector<instruction_ref>& args,
+                                        const std::string& op_name,
+                                        T alpha = 1.0f,
+                                        T beta  = 0.0f)
+{
+    if(!(op_name == "dot" or op_name == "quant_dot"))
+        MIGRAPHX_THROW("Alpha and Beta calculation can be only be applied to either dot or "
+                       "quant_dot operator");
+
+    auto l1       = args[0];
+    auto l2       = args[1];
```

Comment:
What is `l1` and what is `l2`? Can you use better names? Maybe `arg0` and `arg1`?

Reply:
Maybe name "a" and "b" since we usually say a * b in matrix multiplication.

Reply:
renamed.